Serilog.Sinks.Unity3D/Assets/Serilog.Sinks.Unity3D/UnitySinkExtensions.cs

```diff
@@ -52,12 +53,23 @@ public static class UnitySinkExtensions
             this LoggerSinkConfiguration sinkConfiguration,
             ITextFormatter formatter,
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
-            LoggingLevelSwitch levelSwitch = null)
+            LoggingLevelSwitch levelSwitch = null,
+            UnityEngine.ILogger logger = null)
```

Comment:
If we add an ILogger parameter to this method, then why not support ILogger on the other method too?